mongo/integration/unified/schema_version.go

```diff
@@ -16,7 +16,8 @@ import (
 
 var (
 	supportedSchemaVersions = map[int]string{
-		1: "1.3",
+		// We do not fully support the 1.5 schema, but we need 1.5 to test with observeSensitiveCommands.
+		1: "1.5",
```

Comment:
Since censoring security-sensitive commands in command monitoring is a high priority, this was the [suggested way](https://github.com/mongodb/specifications/pull/1015#discussion_r649552452) to run the new 1.5 tests.